### PR TITLE
Updated max_matrix_locations description.

### PR DESCRIPTION
**Pull request by mesozoic-drones**
# Issue

https://github.com/valhalla/valhalla/issues/2982

## Tasklist

This PR needs no tests, other docs updates etc.

## Requirements / Relations

No links or requirements.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,6 +36,7 @@
    * FIXED: Add missing GEOS:GEOS dep to mjolnir target [#2901](https://github.com/valhalla/valhalla/pull/2901)
 
 * **Enhancement**
+   * Updated max_matrix_locations parameter description. [#2983](https://github.com/valhalla/valhalla/pull/2983)
```
Comment (mandeepsandhu) [inline CHANGELOG.md:39]: newest changes are towards the bottom of the list.

Comment (mandeepsandhu) [reply CHANGELOG.md:39]: On second thoughts this change probably does not need a changelog entry since its trivial.

Comment (mesozoic-drones) [reply CHANGELOG.md]: Moved new line to the bottom.

A also thought that this is a tiny update so it doesn't need to go to changelog. But without changelog entry this PR doesn't pass Changelog Validation ci test.

Comment (nilsnolde) [reply CHANGELOG.md]: Don't worry about CI failing for that, it's more meant as a gentle reminder to review whether a changelog entry is needed or not. I'd also say this is one of the cases where that's a 'not'.

Comment (mesozoic-drones) [reply CHANGELOG.md]: Thank you for explanation) I removed new line from the Changelog.

**Review by nilsnolde — APPROVED** [review]
Thanks! No code changes so I stopped all CI.

**Comment by nilsnolde** [comment]
argh.. of course not mergable without CI.. really gotta ship #2793.. somehow I'd like to move to GH actions before though, so easy there to exclude file patterns!

**Comment by mesozoic-drones** [comment]
> argh.. of course not mergable without CI.. really gotta ship #2793.. somehow I'd like to move to GH actions before though, so easy there to exclude file patterns! Also, the close/open PR trick works for action to re-run..

So should I add new line to the changelog?

**Comment by nilsnolde** [comment]
> So should I add new line to the changelog?

All good, that check is not mandatory, the circle CI ones are mostly.

**Pull request merged**